examples/font-info.rs

```diff
@@ -201,3 +115,15 @@ fn print_opentype_layout(name: &str, table: &ttf_parser::opentype_layout::Layout
         println!("    {}", feature);
     }
 }
+
+fn print_opentype_style_attributes(table: &ttf_parser::stat::Table) {
```

Comment:
No need for this anymore since you have extracted the code anyway. We can keep it main.